packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -25,7 +25,13 @@ export class Popup extends BaseComponent<IPopupProps, IPopupState> {
   }
 
   public componentWillMount(): void {
-    this._originalFocusedElement = getDocument()!.activeElement as HTMLElement;
+    let tempActive: HTMLElement = getDocument()!.activeElement as HTMLElement;
```

Comment:
can we please avoid the use of `!` assert?

Reply:
@aftab-hassan I'm still unsure if this fix / behavior is correct since I don't know this component or scenario well.

If this fix is correct, perhaps the code can be slightly refactored to reduce the usage of `!` and casts to the following or similar:

```diff
diff --git a/packages/office-ui-fabric-react/src/components/Popup/Popup.tsx b/packages/office-ui-fabric-react/src/components/Popup/Popup.tsx
index 5c004ab87..fa320d7c0 100644
--- a/packages/office-ui-fabric-react/src/components/Popup/Popup.tsx
+++ b/packages/office-ui-fabric-react/src/components/Popup/Popup.tsx
@@ -25,15 +25,22 @@ export class Popup extends BaseComponent<IPopupProps, IPopupState> {
   }
 
   public componentWillMount(): void {
-    let tempActive: HTMLElement = getDocument()!.activeElement as HTMLElement;
+    let tempActive: Element | null = null;
+    let document: Document | undefined = getDocument();
 
-    while (tempActive instanceof HTMLIFrameElement) {
-      if (tempActive !== null && tempActive.contentDocument !== null) {
-        tempActive = tempActive.contentDocument.activeElement as HTMLIFrameElement;
+    if (document) {
+      tempActive = document.activeElement;
+    }
+
+    while (tempActive !== null && tempActive instanceof HTMLIFrameElement) {
+      if (tempActive.contentDocument !== null && tempActive.contentDocument.activeElement !== null) {
+        tempActive = tempActive.contentDocument.activeElement;
       }
     }
 
-    this._originalFocusedElement = tempActive as HTMLElement;
+    if (tempActive !== null) {
+      this._originalFocusedElement = tempActive as HTMLElement;
+    }
   }
 
   public componentDidMount(): void {
```